src/imitation/scripts/config/train_preference_comparisons.py

```diff
@@ -60,6 +61,31 @@ def train_defaults():
     checkpoint_interval = 0  # Num epochs between saving (<0 disables, =0 final only)
     query_schedule = "hyperbolic"
 
+    # Whether to use the PEBBLE algorithm (https://arxiv.org/abs/2106.05091)
+    pebble_enabled = False
+    unsupervised_agent_pretrain_frac = 0.0
+
+
+@train_preference_comparisons_ex.named_config
+def pebble():
+    # fraction of total_timesteps for training before preference gathering
+    pebble_enabled = True
+    unsupervised_agent_pretrain_frac = 0.05
+    pebble_nearest_neighbor_k = 5
+
+    rl = {
+        "rl_cls": sb3.SAC,
+        "batch_size": 256,  # batch size for RL algorithm
+        "rl_kwargs": {"batch_size": None},  # make sure to set batch size to None
+    }
+    train = {
+        "policy_cls": base.SAC1024Policy,  # noqa: F841
+    }
+    common = {"env_name": "MountainCarContinuous-v0"}
+    allow_variable_horizon = True
```

Comment:
I understand, but this is related to the MountainCarContinuous-v0 environment. As far as I can see, the same pattern is used in other places (e.g., line 93). 
Is there a way to avoid this without using a different environment? I noticed there are things like `AbsorbAfterDoneWrapper` but I didn't dig deeper.

Reply:
Resolved together with the point above.